docs/2-advanced/01-configuration.md

```diff
@@ -20,6 +20,7 @@ There are several configuration settings that affect all UI5 Web Components glob
 <a name="theme"></a>
 
 The `theme` setting values above are the technical names of the supported themes:
+- The `sap_horizon` is known as `Horizon` and it's the latest - currently as "preview".
```

Comment:
@LilyanaOviPe  please suggest better way to state the sap_horizon is the latest theme added, but it's available as "preview" (meaning not final).

Reply:
- The `sap_horizon` is known as `Horizon` and it's the latest theme currently available as a preview version.